frankenphp.c

```diff
@@ -459,6 +459,19 @@ static char* frankenphp_read_cookies(void)
 	return ctx->cookie_data;
 }
 
+void frankenphp_register_bulk_variables(char **variables, int size, zval *track_vars_array)
+{
+	char * k;
+
+	for (int i = 0; i < size; i++)
+	{
+		if (i%2 == 0) {
+			k = variables[i];
+			continue;
+		}
+	}
+}
```

Comment:
```suggestion
void frankenphp_register_bulk_variables(char **variables, int size, zval *track_vars_array)
{
	for (int i = 0; i < size; i++)
	{
		if (i%2 == 1) php_register_variable(variables[i-1], variables[i], track_vars_array);
	}
}
```

This fixes the missing call to `php_register_variable()` and prevents a useless assignation.